assets/js/modules/analytics/components/dashboard/DashboardOverallPageMetricsWidget/utils.js

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
Thanks, @techanvil. This almost looks good to me, but could you please move these `utils.js` files into the `assets/js/modules/analytics/util` folder and revert widgets to be in single files again? Once this is done, I'll approve this PR.

Reply:
Ok, sure, that's done. I moved the GA4 version into `assets/js/modules/analytics-4/utils/`, though.